lib/interpret_and_optimize/entities/subclasses/pb_intermediate_node.dart

```diff
@@ -193,11 +194,30 @@ abstract class PBIntermediateNode
 
   void mapRawChildren(Map<String, dynamic> json, PBIntermediateTree tree) {
     var rawChildren = json['children'] as List;
+    var parentConstraints = json['constraints'];
     rawChildren?.forEach((rawChild) {
       if (rawChild != null) {
+        /// Child inherit Parent's constraints if they are fixed
```

Comment:
These should take place in the Positional Alignment rules, otherwise we are causing Shotgun Surgery